.changeset/hip-glasses-grin.md

```diff
@@ -0,0 +1,5 @@
+---
+'@firebase/database': minor
+---
+
+Add `startAfter` for RTDB queries
```

Comment:
Do you want to add a bit of an explanation on what this is used for? This is the space for you to sell this to our users, especially those that struggle with pagination.